addons/medical/stringtable.xml

```diff
@@ -123,7 +123,7 @@
         </Key>
         <Key ID="STR_ACE_Medical_Apply_Tourniquet">
             <English>Apply Tourniquet</English>
-            <German>Tourniquet anwenden</German>
+            <German>Druckverband anwenden</German>
```

Comment:
Ein Druckverband ist etwas gänzlich anderes als ein Tourniquet. EIn Tourniquet hätte eine entsprechende Übersetzung in einer Aderpresse. Innerhalb der Bundeswehr ist das Wort Tourniquet geläufig.